eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -177,6 +177,45 @@ function Update-CIYmlFile() {
     }
 }
 
+function RegisterMgmtSDKToMgmtCoreClient () {
```

Comment:
We should look at sharing instead of duplicating this implementation from https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Update-Mgmt-CI.ps1

Reply:
Hello @weshaggard , I updated to use the common function in Update-Mgmt-CI.ps1.